utils/src/server/rpc_module.rs

```diff
@@ -33,7 +33,7 @@ pub type MethodSink = mpsc::UnboundedSender<String>;
 
 type Subscribers = Arc<Mutex<FxHashMap<UniqueSubscriptionEntry, (MethodSink, oneshot::Receiver<()>)>>>;
 
-/// Represent a unique subscription entry based on [`SubscriptionID`] and [`ConnectionID`].
+/// Represent a unique subscription entry based on SubscriptionID and ConnectionID.
```

Comment:
The rustdoc complaint was caused by a misspelled type name I think, `ID` vs `Id`. So think this should work:

```
Represent a unique subscription entry based on [`SubscriptionId`] and [`ConnectionId`].
```

Reply:
thanks changed it